docs/manual-instrumentation.md

```diff
@@ -0,0 +1,43 @@
+# Manually instrument a .NET application
+
+The auto-instrumentation provides a base you can build on by adding your own
+custom instrumentation. By using both instrumentation approaches,
+you'll be able to present a more detailed representation of the logic
+and functionality of your application, clients, and framework.
+
+## Instrument using System.Diagnostics API
+
+For the list of steps required, see [link](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md).
+
+## Instrument using OpenTracing API
```

Comment:
I think we should add a hyperlink in `migrating.md` to this header.

Reply:
Couldn’t we move the section to the migration section? It feels a bit strange to me that someone would use OpenTracing when a new Otel rostro is available. At the very minimum we should explain why would someone use OT here.